file_and_modules.md

```diff
@@ -41,7 +41,31 @@
 * We can import a module using `import` keyword.
 * If it is not available it will give import error.
 * dir() method tells what the attribute provides.
+* What's [The Zen of Python] (https://www.python.org/dev/peps/pep-0020/) ? 
```

Comment:
* What's [The Zen of Python] (https://www.python.org/dev/peps/pep-0020/) ? 

It's getting rendered like this, can you please fix it?

Reply:
@gauravsitlani yup... please remove `white-space`.